app/blueprints/api/v4/race_blueprint.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V4::RaceBlueprint < Blueprinter::Base
+  fields :id, :status_text, :visibility, :notes, :started_at, :created_at, :updated_at
+
+  association :entrants, blueprint: Api::V4::EntrantBlueprint
+
+  view :standard do
+    association :category, blueprint: Api::V4::CategoryBlueprint
+  end
+
+  view :bingo do
+    field :card do |race, _|
+      race.started? ? race.card : nil
+    end
+
+    association :game, blueprint: Api::V4::GameBlueprint
+  end
+
+  view :randomizer do
+    field :seed do |race, _|
+      race.started? ? race.seed : nil
```

Comment:
As far as I'm aware, you withhold the seed/card so that people cannot start routing or planning before the race starts.  I'm not sure if there is a subset that would rather share it as bingos and randomizers were never really my thing.

Since our system is more focused on the honor system it would probably be fine have it revealed from race creation though.  We can always add in later some kind of time system that the card is revealed X seconds from start of race if that is preferable.  Would you prefer that it is shown from the beginning then?

Reply:
Yeah I say let's first do the honor system then adjust if needed.